README.md

```diff
@@ -5,13 +5,15 @@ Wiki for the Pharo programming language and development environment.
 The goals of the wiki are explained in the [Manifest](MANIFEST.md) and contribution guidelines in the [Contribution guide](CONTRIBUTION.md).
 
 ## Contents
-- [Beginners](#beginners)
-- [General](#general)
-- [Pharo projects](#pharo-projects)
-- [External projects](#external-projects)
-  * [Data exchange](#data-exchange)
-- [Migration guidelines](#migration-guidelines)
-- [External ressources](#external-ressources)
+- [Pharo wiki](#pharo-wiki)
```

Comment:
I am not sure that adding the title of level 1 really makes sense. 

In most pages, we went with the convention of starting with the second level.

Reply:
That was likely the extension that did it -- not sure why (now I'm curious if I can turn that function off).